### PR TITLE
Andrygzt/edit app fixes

### PR DESCRIPTION
**Pull request by andrygzt**
1. Prisma client unable to run in browser fixed.
2. Edit app problem fixed from profile component.
3. Save button in editing app successfully save changes.

### CODE REVIEW

File: components/AppExtentionPage.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import AppCard from "./AppCard";
```
Comment (VidaMaleki) [inline components/AppExtentionPage.tsx]: Why did you make it lowercase if the file starts with uppercase?

File: components/AppExtentionPage.tsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
+import AppCard from "./appCard";
```
Comment (VidaMaleki) [inline components/AppExtentionPage.tsx]: same to this one

File: components/appCard.tsx [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import AppDetailsPopup from './AppDetailsPopUp';
```
Comment (VidaMaleki) [inline components/appCard.tsx]: We will get error later if you change to lowercase

File: prisma/migrations/20230710222340_init/migration.sql [hunk 4]
```diff
@@ -0,0 +1,35 @@
+-- CreateTable
+CREATE TABLE "Developer" (
+    "id" SERIAL NOT NULL,
```
Comment (VidaMaleki) [inline prisma/migrations/20230710222340_init/migration.sql:3]: Can I know why did you remove auto-increment from Id?

File: prisma/migrations/20230710222340_init/migration.sql [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    "fullName" TEXT NOT NULL,
+    "email" TEXT NOT NULL,
```
Comment (VidaMaleki) [inline prisma/migrations/20230710222340_init/migration.sql:5]: Did you read Prisma? In Prisma you don't need to write your code like this

Comment (VidaMaleki) [inline prisma/migrations/20230710222340_init/migration.sql:5]: Did you read Prisma? In Prisma, you don't need to write your code like this

**Comment by VidaMaleki** [comment]
I will add rest of edit app inputs later but I ran the code was working.

**Review by VidaMaleki — APPROVED** [review]

File: prisma/migrations/20230710222340_init/migration.sql (hunk 4 above)
Comment (andrygzt) [reply prisma/migrations/20230710222340_init/migration.sql:3]: Was settle by prisma dev commands

File: prisma/migrations/20230710222340_init/migration.sql (hunk 5 above)
Comment (andrygzt) [reply prisma/migrations/20230710222340_init/migration.sql:5]: Was settle by prisma dev commands

Comment (andrygzt) [reply prisma/migrations/20230710222340_init/migration.sql:5]: Was settle by prisma dev commands

File: lib/db.ts [hunk 6]
```diff
@@ -7,18 +7,20 @@ declare global {
 let prisma: PrismaClient;
 
 async function connect() {
-    if (process.env.NODE_ENV === 'production') {
-        prisma = new PrismaClient();
-    } else {
-        if (!global.cachedPrisma) {
-        global.cachedPrisma = new PrismaClient();
+    if (typeof window === "undefined") {
```
Comment (andrygzt) [inline lib/db.ts:10]: statement is needed as PrismaClient must be run in the server considering browser is not able to run it.
